src/pages/clubs/redux/index.tsx

```diff
@@ -59,12 +59,23 @@ const reduxClub: React.FC<any> = ({data}) => {
 
 export default reduxClub
 
+export const reduxClubQuery = groq`*[_type == 'resource' && slug.current == "build-business-oriented-portfolio"][0]{
+  "redux": resources[0].resources[slug.current == "redux"][0]{
+      title,
+      subTitle,
+      description,
+      "slug": slug.current,
+      image,
+      summary,
+    }
+}`
+
 export async function getStaticProps() {
-  const data = await sanityClient.fetch(reduxClubQuery)
+  const {redux} = await sanityClient.fetch(reduxClubQuery)
```

Comment:
just give the component what it needs instead of generic `data`

Reply:
A good point hadn't thought about this. Or just assumed that we would be querying only the data that was needed